src/admin/components/ResourceTabs/PolygonReviewTab/components/PolygonStatus/StatusDisplay.tsx

```diff
@@ -174,10 +203,19 @@ const StatusDisplay = ({ titleStatus = "Polygon", mutate, refresh, name, record,
               type: "error",
               title: "Error!"
             });
+            setTimeout(() => {
+              setNotificationStatus({
+                open: false,
+                message: "",
+                type: "error",
+                title: "Error!"
+              });
+            }, 3000);
+            console.error(e);
           } finally {
-            refresh();
-            reloadEntity();
-            closeModal();
+            refresh && refresh();
+            reloadEntity && reloadEntity();
```

Comment:
Please use safe navigation instead of boolean expression:
```
refresh?.();
reloadEntity?.();
```

Reply:
Done